README.md

```diff
@@ -30,48 +12,134 @@ Command for running postfix docker container:
 ```
 docker run -it -e MYHOSTNAME=localhost \
     -p 25:10025\
-    -v $(pwd)/tmp/postfix:/var/spool/postfix \
-    -v $(pwd)/tmp/mail:/var/spool/mail postfix
+    -v /var/spool/postfix:/var/spool/postfix \
+    -v /var/spool/mail:/var/spool/mail postfix
+    -d {{ spec.image_registry }}
 ```
 
 ## How to use the container over standard TLS
 
-Dockerfile for TLS port is Dockerfile.TLS.
-
 Command for running postfix docker container:
 ```
-docker run -it -e DEBUG_MODE=yes -e MYHOSTNAME=localhost \
+docker run -it -e ENABLE_TLS -e DEBUG_MODE=yes -e MYHOSTNAME=localhost \
     -p 25:10025 \
-    postfix
+    postfix -d {{ spec.image_registry }}
 ```
 
 Environment variable DEBUG_MODE is used for debugging proposes
 from Postfix and dovecot point of views.
 
-## How to use the container with IMAP with TLS enabled (dovecot and Postfix)
-
-Dockerfile which uses IMAP is Dockerfile.IMAP.
+## How to use the container with IMAP.
 
 Command for running postfix docker container:
 ```
-docker run -it -e MYHOSTNAME=localhost -e DEBUG_MODE \
-    -p 25:10025 -p 587:10587 -p 143:10143 \
-    postfix
+docker run -it -e ENABLE_IMAP -e MYHOSTNAME=localhost
+    -e DEBUG_MODE \
+    -p 587:10587 \
+    -v <path_tocertificates>:/etc/postfix/certs \
+    postfix -d {{ spec.image_registry }}
 ```
 
 Environment variable DEBUG_MODE is used for debugging proposes
 from Postfix and dovecot point of views.
 
-Docker container does NOT share user from HOST system. Therefore
-I prefer to create users in container. This is not yet automated. Will be SOON.
+# How to generate self signed SSL key for {{ spec.envvars.name }}
```

Comment:
This section is awesome! Could you please be more verbose and precise what the steps do and why users need to perform them? That should help beginners a lot.